domains/alexandra.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "alexxandraSalazar",
+    "email": "alexandrasalazar121205@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://alexandrasalazar.vercel.app/"
```

CNAME can't contain protocol or slashes
```suggestion
    "CNAME": "alexandrasalazar.vercel.app"
```